scss/utilities/icon-mixin.scss

```diff
@@ -51,6 +51,7 @@
   height: $size;
   color: $default-color;
   box-shadow: unquote($ret);
+  transform: rotate(0.0001deg);
```

Comment:
Conflict with other styles
https://github.com/nostalgic-css/NES.css/blob/b969073d7e35db54bef5668a68d7bcde13e7a077/scss/icons/icons.scss#L41-L45

Reply:
Hi, I've updated that commit message now, it should be ready to merge 👍